mappings/net/minecraft/test/StartupParameter.mapping

```diff
@@ -1,4 +1,258 @@
+		ARG 4 y
+		ARG 6 z
+	METHOD method_36006 spawnMob (Lnet/minecraft/class_1299;FFF)Lnet/minecraft/class_1308;
+		ARG 1 type
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_36007 spawnMob (Lnet/minecraft/class_1299;III)Lnet/minecraft/class_1308;
+		ARG 1 type
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_36008 spawnMob (Lnet/minecraft/class_1299;Lnet/minecraft/class_243;)Lnet/minecraft/class_1308;
+		ARG 1 type
+		ARG 2 vec3d
+	METHOD method_36009 spawnMob (Lnet/minecraft/class_1299;Lnet/minecraft/class_2338;)Lnet/minecraft/class_1308;
+		ARG 1 type
+		ARG 2 pos
+	METHOD method_36011 dontExpectBlock (Lnet/minecraft/class_2248;III)V
+		ARG 1 block
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_36012 dontExpectBlock (Lnet/minecraft/class_2248;Lnet/minecraft/class_2338;)V
+		ARG 1 block
+		ARG 2 pos
+	METHOD method_36013 (Lnet/minecraft/class_243;Lnet/minecraft/class_1297;)Z
+		ARG 1 entity
+	METHOD method_36014 getBlockEntity (Lnet/minecraft/class_2338;)Lnet/minecraft/class_2586;
+		ARG 1 pos
+	METHOD method_36017 checkBlockState (Lnet/minecraft/class_2338;Ljava/util/function/Predicate;Ljava/util/function/Supplier;)V
+		ARG 1 pos
+		ARG 2 predicate
+		ARG 3 errorMessageSupplier
+	METHOD method_36018 addInstantFinalTask (Ljava/lang/Runnable;)V
+		ARG 1 runnable
+	METHOD method_36021 createMockPlayer ()Lnet/minecraft/class_1657;
+	METHOD method_36022 expectEntityAt (Lnet/minecraft/class_1299;III)V
+		ARG 1 type
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_36023 expectEntityAt (Lnet/minecraft/class_1299;Lnet/minecraft/class_2338;)V
+		ARG 1 type
+		ARG 2 pos
+	METHOD method_36024 expectBlockAtEnd (Lnet/minecraft/class_2248;III)V
+		ARG 1 block
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_36025 expectBlockAtEnd (Lnet/minecraft/class_2248;Lnet/minecraft/class_2338;)V
+		ARG 1 block
+		ARG 2 pos
+	METHOD method_36026 pushButton (Lnet/minecraft/class_2338;)V
+		ARG 1 pos
+	METHOD method_36030 useNightTime ()V
+	METHOD method_36031 dontExpectEntityAt (Lnet/minecraft/class_1299;III)V
+		ARG 1 type
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_36032 dontExpectEntityAt (Lnet/minecraft/class_1299;Lnet/minecraft/class_2338;)V
+		ARG 1 type
+		ARG 2 pos
+	METHOD method_36034 useBlock (Lnet/minecraft/class_2338;)V
+		ARG 1 pos
+	METHOD method_36036 complete ()V
+	METHOD method_36037 (Lnet/minecraft/class_1299;III)V
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_36038 (Lnet/minecraft/class_1299;Lnet/minecraft/class_2338;)V
+		ARG 1 type
+		ARG 2 pos
+	METHOD method_36039 toggleLever (Lnet/minecraft/class_2338;)V
+		ARG 1 pos
+	METHOD method_36042 (Lnet/minecraft/class_1299;III)V
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_36043 (Lnet/minecraft/class_1299;Lnet/minecraft/class_2338;)V
+		ARG 2 pos
+	METHOD method_36044 removeBlock (Lnet/minecraft/class_2338;)V
+		ARG 1 pos
+	METHOD method_36045 getTick ()J
+	METHOD method_36047 expectEmptyContainer (Lnet/minecraft/class_2338;)V
+		ARG 1 pos
+	METHOD method_36048 checkIfHasFinalClause ()V
```

Comment:
```suggestion
	METHOD method_36048 markFinalClause ()V
```
A bit clunky, but more accurate than `checkIfHas`. This checks if there is already a final clause and then throws an `IllegalStateException`, otherwise sets `hasFinalClause` to true.